packages/valory/skills/abstract_round_abci/test_tools/abci_app.py

```diff
@@ -77,10 +77,6 @@ class ConcreteTerminationRoundA(_ConcreteRound):
 
     allowed_tx_type = "payload_termination_a"
 
-    def end_block(self) -> Tuple[MagicMock, MagicMock]:
-        """End block."""
-        return MagicMock(), MagicMock()
-
```

Comment:
was the intention of previous PR #1633, see [this comment](https://github.com/valory-xyz/open-autonomy/pull/1633#discussion_r1043401515)